src/card/card.jsx

```diff
@@ -31,6 +32,10 @@ const Card = React.createClass({
       }
       if (this.state.expanded === false && currentChild.props.expandable === true)
         return;
+      if (currentChild.props.actAsExpander === true) {
+        currentChild.props.onTouchTap = this._onExpandable;
```

Comment:
@hugo-agbonon this will not work with React 0.14. Props can no longer be mutated.. see here: https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html#breaking-changes

I'll try to fix this.